Sources/Rendering/Core/Prop/index.d.ts

```diff
@@ -230,11 +230,25 @@ export interface vtkProp extends vtkObject {
 
     /**
      * This is used for culling and is a number between 0 and 1. It is used to create the allocated render time value.
-     * @param {Number} renderTimeMultiplier 
+     * @param {Number} renderTimeMultiplier
      */
     setRenderTimeMultiplier(renderTimeMultiplier: number): boolean;
 
     /**
+     * Indicate that this prop's data should be in world coordinates.
```

Comment:
what about the "userMatrix" ? in what coordinate system is it ?

Reply:
who knows, only the user. But IIRC the user matrix just gets multiplied into everything else (or it replaces everything else, I forget which) But when it gets to the mapper the results of that will be interpreted by the mapper as WORLD or DISPLAY.

Reply:
Gotcha, so it might be good to precise in Prop3D/index.d.ts that `origin`, `position`, `orientation` and `userMatrix` are expressed in this coordinate system (world or display).